packages/autoloader/tests/php/test_file_loader.php

```diff
@@ -1,66 +1,80 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName.NotHyphenatedLowerCase
+/**
+ * File loader test suite.
+ *
+ * @package automattic/jetpack-autoloader
+ */
 
-use Automattic\Jetpack\Autoloader as Autoloader;
 use PHPUnit\Framework\TestCase;
 
+/**
+ * Test suite class for the Autoloader part that handles file loading.
+ */
 class WP_Test_File_Loader extends TestCase {
-	static $jetpack_packages_files;
-	function setup() {
+
+	/**
+	 * Setup runs before each test.
+	 */
+	public function setup() {
 		parent::setup();
-		global $jetpack_packages_files;
-		self::$jetpack_packages_files = $jetpack_packages_files;
-		$jetpack_packages_files = array();
-	}
 
-	function tearDown() {
-		parent::tearDown();
-		// re-apply the global
-		global $jetpack_packages_files;
-		$jetpack_packages_files = self::$jetpack_packages_files;
+		spl_autoload_register( 'autoloader' );
```

Comment:
I think we can remove this `spl_autoload_register()` call because these tests just test the 'autoloading' of files. The spl autoloader just autoloads classes.